cms/envs/common.py

```diff
@@ -2265,3 +2248,5 @@
 # .. toggle_tickets: ARCH-1253
 # .. toggle_status: supported
 DISABLE_DEPRECATED_SIGNUP_URL = False
+
+FEATURES['TAHOE_SITES_USE_ORGS_MODELS'] = False
```

Comment:
Please move into `openedx/core/djangoapps/appsembler/settings/settings/test_common.py`. This and all `TAHOE_*` features such as `TAHOE_STUDIO_LOCAL_LOGIN` and `TAHOE_ENABLE_API_DOCS_URLS`.

```suggestion
FEATURES['TAHOE_SITES_USE_ORGS_MODELS'] = False
```

Reply:
Done